Automattic/wp-calypso
# SDK: Namespace CSS

Prefix all CSS selectors with an additional class named after the extension name, as inferred from the path name.

To test: 

* Run `npm run sdk:gutenberg -- --editor-script=client/gutenberg/extensions/editor-notes/index.js` to verify that Gutenberg extension building still works. 
  * Verify that all selectors in `client/gutenberg/extensions/editor-notes/build/editor-notes-editor.js.css` are now prefixed with an additional `.editor-notes` class.
* Verify that Calypso still builds (and runs!) correctly, in different environments:
  * Try `npm start`
  * Try a Docker build of wpcalypso (and production, if possible)
